### PR TITLE
scripts: cloud9-toolkit-install.sh

### PR DESCRIPTION
**Pull request by justinmk3**
## Problem

Testing a Toolkit artifact in Cloud9 requires manual steps.

## Solution

Automate the steps using a script.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloud9-toolkit-install.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# a ZIP (containing a vsix).
+#
+# USAGE:
+#     cloud9-toolkit-install.sh [URL|FILE]
+#     curl -LO https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/cloud9-toolkit-install.sh && bash cloud9-toolkit-install.sh
+# EXAMPLES:
+#     cloud9-toolkit-install.sh https://github.com/aws/aws-toolkit-vscode/releases/download/v1.24.0/aws-toolkit-vscode-1.24.0.vsix
+#     cloud9-toolkit-install.sh toolkit.zip
+
+set -eu
```
Comment (hunterwerlla) [inline cloud9-toolkit-install.sh:15]: We need to use this more often

File: cloud9-toolkit-install.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    mv extension "$TOOLKIT_INSTALL_DIR"
+    _log "Toolkit installed to: $TOOLKIT_INSTALL_DIR"
+    _log "Refresh Cloud9 to load it"
+)
+}
+
+_main
+
+# vim: et sw=4 ts=4
```
Comment (hunterwerlla) [inline cloud9-toolkit-install.sh:112]: remove? This file should get formatted by prettier as well?

**Review by hunterwerlla — APPROVED** [review]

File: cloud9-toolkit-install.sh (hunk 2 above)
Comment (justinmk3) [reply cloud9-toolkit-install.sh:112]: I found https://github.com/prettier/prettier/issues/4555 , seems like prettier doesn't support shell. I wouldn't trust it anyway, parsing bash/csh/zsh/sh is extremely non-trivial.

Comment (justinmk3) [reply cloud9-toolkit-install.sh:112]: - Added `prettier-plugin-sh` in https://github.com/aws/aws-toolkit-vscode/pull/1673/commits/8e37afa7f86b07d38d087fc320414e5895d45e6c 
- see https://github.com/aws/aws-toolkit-vscode/pull/1673/commits/bf15204443a0dfb13255a30b8a3035815d1eed68 for the result (mostly whitespace).

**Review by bryceitoc9 — APPROVED** [review]

**Pull request merged**